internal/generator/backend/template/zkpschemes/plonk/tests/marshal.go.tmpl

```diff
@@ -153,7 +163,7 @@ func randomPoint() curve.G1Affine {
 	return r
 }
 
-func randomScalars( n int) []fr.Element {
+func randomScalars(n int) []fr.Element {
```

Comment:
forgot where this comes from; but with some retrospective, and what happened in `gnark-crypto`; this should be a better randomize function (ie call `SetRandom()` everywhere ? -